src/main/java/com/robinpowered/sdk/model/Identifiable.java

```diff
@@ -0,0 +1,57 @@
+package com.robinpowered.sdk.model;
+
+import com.google.gson.annotations.SerializedName;
+
+import org.joda.time.DateTime;
+
+/**
+ * Interface for identifiers of {@link Device Devices}.
+ */
+public interface Identifiable {
```

Comment:
This term is already used to describe models with a `getId`. Need to think of how we can make these different glossary terms.